src/transport/multicast/read.c

```diff
@@ -38,6 +39,25 @@ int8_t _zp_multicast_read(_z_transport_multicast_t *ztm) {
     return ret;
 }
 
+int _zp_multicast_start_read_task(_z_transport_t *zt, _z_task_attr_t *attr, _z_task_t *task) {
+    // Init memory
+    (void)memset(task, 0, sizeof(_z_task_t));
+    // Attach task
+    zt->_transport._multicast._read_task = task;
+    zt->_transport._multicast._read_task_running = true;
+    // Init task
+    if (_z_task_init(task, attr, _zp_multicast_read_task, &zt->_transport._multicast) != _Z_RES_OK) {
+        zt->_transport._multicast._read_task_running = false;
+        return _Z_ERR_SYSTEM_TASK_FAILED;
+    }
+    return _Z_RES_OK;
+}
+
+int _zp_multicast_stop_read_task(_z_transport_t *zt) {
```

Comment:
As convention, return `int8_t` instead.
This goes back to a previous discussion on having a list of return codes.